src/components/ModelingMachineProvider.tsx

```diff
@@ -751,13 +751,22 @@ export const ModelingMachineProvider = ({
               if (err(varDec)) return reject(new Error('No varDec'))
               const variableName = varDec.node.declaration.id.name
               let isIdentifierUsed = false
-              traverse(newAst, {
-                enter: (node) => {
-                  if (node.type === 'Name' && node.name.name === variableName) {
-                    isIdentifierUsed = true
-                  }
-                },
-              })
+              const isInitAPipe =
+                varDec.node.declaration.init.type === 'PipeExpression'
+              if (isInitAPipe) {
+                isIdentifierUsed = true
```

Comment:
Name is not the best now that we're check for use in general not only if it's the identifier but 🤷 too late now.